_config.yml

```diff
@@ -176,3 +173,42 @@ exclude:
 #    max_zoom: 10
 #    subfolder: my-geojson-subfolder
 #    label: My map layer label (can be a translation key)
+
+# Recommend accessibility improvements.
+accessible_tabs: true
+accessible_charts: true
+
+# Specify any custom javascript files.
+custom_js:
+  - /assets/js/custom.js
+
+# Specify the format of the language toggles. Choices are:
+# "dropdown", "links", or "none"
+header_language_toggle: dropdown
+footer_language_toggle: none
+
+# Specify whether the Series disaggregation should be a toggle (like units).
+series_toggle: true
+
+# Speficy the type of header to display. Choices are:
+# "header-default.html", "header-menu-left-aligned.html", or any custom file
+header:
+  include: header-menu-left-aligned.html
```

Comment:
Since this is default, should `header-default.html` be the example option?

Reply:
@LucyGwilliamAdmin Actually I realized that is not going in until 1.3.0, so I removed that section. However when we get to that point, I think we should put which ever one we recommend. I think we recommend the left-aligned version, so that's why I added it.

Reply:
Ah right - that makes sense. I wondered if it was something like that